frontend/src/widgets/TaskCard/taskcard.jsx

```diff
@@ -0,0 +1,30 @@
+/** @jsxImportSource @emotion/react */
+
+import Card from 'components/Card';
+
+// Style
+import * as style from './taskcard-styles';
+
+export default function TaskCard({ children, title, body, assignee, tag, ...passedProps }) {
+  return (
+    <div css={style.taskCard} {...passedProps}>
+      <Card>
+        <div css={style.taskCard_header}>
+          <p css={style.taskCard_title}>
+            {title}
+          </p>
+          <div css={style.taskCard_image}>
+            <img src={assignee} alt="user" css={style.taskCard_imageContainer}/>
```

Comment:
in practice, an assignee would be a javascript object and not an image so better if i simulate nga ang assignee is an object with an image, so ang i butang na diri kay `src={assignee.image}`, another method would be to rename assignee to assigneeImage, which would better describe it